mp/game/momentum/scripts/HudAnimations.txt

```diff
@@ -929,4 +929,9 @@ event HideVoteBackgrounds
 	Animate		LabelOption3	Alpha	255 Linear	0.0 0.001
 	Animate		LabelOption4	Alpha	255 Linear	0.0 0.001
 	Animate		LabelOption5	Alpha	255 Linear	0.0 0.001
+}
+
+event TimerColorReset
+{
+	Animate HudTimer StatusColor "Mom.Panel.Fg" Spline 0.3 0.3
 }
```

Comment:
This should be next to the TimerFailStart event instead of at the bottom of the file. You can also use this animation from the TimerColor animation event, like so:

```
event TimerFailStart
{
    Animate HudTimer StatusColor "255 0 0 255" Spline 0.0 0.3
    RunEvent TimerColorReset 0.3
}

event TimerColorReset
{
	Animate HudTimer StatusColor "Mom.Panel.Fg" Spline 0.0 0.3
}
```

The first parameter after the type of animation is the delay in which it takes to do the command. For the case of the reset animation, you'd want `0.0`, to start it immediately, and `0.3` to fade it back to the color in 0.3 seconds. You can also find more info here: https://developer.valvesoftware.com/wiki/Understanding_VGUI2_Animation